src/Bathymetry.jl

```diff
@@ -161,30 +123,12 @@ function regrid_bathymetry(target_grid;
         z_data[land] .= height_above_water
     end
 
-    # Infer the "native grid" of the bathymetry data and make a bathymetry field.
-    Δλ = λ_data[2] - λ_data[1]
-    Δφ = φ_data[2] - φ_data[1]
-
-    λ₁_data = convert(Float64, λ_data[1]   - Δλ / 2)
-    λ₂_data = convert(Float64, λ_data[end] + Δλ / 2)
-    φ₁_data = convert(Float64, φ_data[1]   - Δφ / 2)
-    φ₂_data = convert(Float64, φ_data[end] + Δφ / 2)
-
-    Nxn = length(λ_data)
-    Nyn = length(φ_data)
-    Nzn = 1
-
-    native_grid = LatitudeLongitudeGrid(arch, Float32;
-                                        size = (Nxn, Nyn, Nzn),
-                                        latitude  = (φ₁_data, φ₂_data),
-                                        longitude = (λ₁_data, λ₂_data),
-                                        z = (0, 1),
-                                        halo = (10, 10, 1))
-
     native_z = Field{Center, Center, Nothing}(native_grid)
     set!(native_z, z_data)
     fill_halo_regions!(native_z)
 
+    ### We still need to fix the grids so they are the same - how did Greg do this?
```

Comment:
@glwagner I have defined a `native_grid` and `target_grid` - I just need to get them to align without using `circshift` or similar. I couldn't figure out how you did this for the EN4/ECCO datasets?

Reply:
We do not need to align grids if they are in the same longitude range. Oceananigans will interpolate the data taking into account the circularity of the longitude dimension. 
For example, for Oceananigans, a longitude range of (-180, 180) is equivalent to a range (180, 360) concatenated to (0, 180). Another example: a range (700, 720) is equivalent to (340, 360) and so on...

Reply:
We don't use circshift anymore because it is not a generic solution. I don't think you need a custom native_grid implementation for this.

Reply:
Yes, I didn't use a custom `native_grid` implementation now